MyToDo/App/Category+CoreDataProperties.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  Category+CoreDataProperties.swift
+//  MyToDo
+//
+//  Created by Ivan Ruiz Monjo on 20/2/21.
+//
+//
+
+import Foundation
+import CoreData
+
+
+extension Category {
+
+    @nonobjc public class func fetchRequest() -> NSFetchRequest<Category> {
+        return NSFetchRequest<Category>(entityName: "Category")
+    }
+
+    @NSManaged public var imageName: String?
+    @NSManaged public var title: String
+    @NSManaged public var todos: NSSet?
+
+}
+
+// MARK: Generated accessors for todos
+extension Category {
```

Comment:
These accessors are not needed since you can mutate the `todos` directly, I think these are somewhat of a leftover from objective-c where `NSSet` isn't homogenous like Swift's `Set`. These methods would provide some primitive type safety